openmcyclus/depletion.py

```diff
@@ -165,16 +162,15 @@ def get_spent_comps(self, material_ids, path):
             list of the compositions from the OpenMC model
         '''
         results = od.Results(path + "depletion_results.h5")
+        nuclides = od.MicroXS.from_csv(path+"micro_xs.csv").nuclides
         spent_comps = []
-        for index, material_id in enumerate(material_ids):
-            spent_comp = results[-1].get_material(str(material_id))
-
+        for material_id in material_ids:
             comp = {}
-            for nuclide in spent_comp.nuclides:
-                if nuclide.percent < 1e-5:
+            for nuclide in nuclides:
+                Z, A, m = openmc.data.zam(nuclide)
+                mass = results.get_mass(str(material_id), nuclide)[-1][-1]
+                if mass <= 1e-10:
```

Comment:
Careful here! Some libraries have funkiness with the ZAM for Am242_m1 and Am242.

Reply:
They do? Any suggestions on how to safeguard that? I thought this was a pretty standard format.

Reply:
I think OpenMC treates the ZAM for Am242 and Am242_m1 in an intuitive way. If you were using raw ACE or ENDF evaluations, this would be something you need to consider.

Reply:
This should just be for processing what OpenMC spits out, so as long as OpenMC is consistent with itself we should be good.